### PR TITLE
Add fibonacci in kotlin

### PR DESCRIPTION
**Pull request by nlaert**

### CODE REVIEW

File: kotlin_fibonacci.kt [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import java.math.BigInteger
+import java.util.Scanner
+
+
+fun main(args: Array<String>) {
```
Comment (fharookshaik) [inline kotlin_fibonacci.kt:5]: Codacy found an issue: [UndocumentedPublicFunction - [main] at /src/kotlin_fibonacci.kt:5:5](https://app.codacy.com/gh/fharookshaik/fibonacci-series/pullRequest?prid=6356879)

**Comment by fharookshaik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/fharookshaik/fibonacci-series/pullRequest?prid=6356879&bid=20637144)

**Comment by fharookshaik** [comment]
Some things regarding contributing has changed. Please read them carefully once again in the Contributing.md doc.
You're just another step away for your contribution,

Happy Coding!!
Happy Contribution!!

**Comment by nlaert** [comment]
Hello, here are the print screens as asked. Had to split in 2 in order to get 100 members.
Also noticed there were an error in the logic, which I fixed :)
Screenshot 2020-10-18 at 14 51 49
Screenshot 2020-10-18 at 14 52 31

**Comment by fharookshaik** [comment]
Great! Thank you for your enormous contribution.
Feel free to add your name in the contributors.md doc and add the language under the languages.md doc.

Once these are done just comment here. I'm ready to accept your PR.

Once again, Thanks for your sincere effort.

**Pull request merged**